ext/opentelemetry-ext-otcollector/README.rst

```diff
@@ -95,4 +95,4 @@ References
 ----------
 
 * `OpenTelemetry Collector <https://github.com/open-telemetry/opentelemetry-collector/>`_
-* `OpenTelemetry Project <https://opentelemetry.io/>`_
+* `OpenTelemetry <https://opentelemetry.io/>`_
```

Comment:
Is it actually fixing a syntax error?

Reply:
OpenTelemetry Project is the actual thing we use in other readme files

Reply:
Actually a syntax error. Either this line or L22 needs to change.

```
% python -m readme_renderer ext/opentelemetry-ext-otcollector/README.rst
<string>:22: (ERROR/3) Unknown target name: "opentelemetry".
%
```

Reply:
Right!